icons/zoho.svg

```diff
@@ -0,0 +1 @@
2.263.363.027.026.744-.058 2.282-.267 1.234-.168 2.254-.295 2.267-.282.024.023.722 4.834.704 4.852-.027.026-4.913.691-4.933.67-.013-.013-.138-.828-.278-1.81-.14-.984-.26-1.816-.266-1.85-.01-.047-1.26 2.716-1.617 3.573-.041.098-.093.178-.116.178-.023 0-1.022-.44-2.22-.98zm5.894.476a.433.433 0 00.246-.386c0-.058-.045-.405-.1-.77-.056-.368-.093-.676-.082-.686.026-.026 1.383-.22 1.402-.2.009.007.065.34.126.736.066.433.135.76.171.815.151.231.479.26.667.057.086-.087.107-.14.107-.277 0-.203-.468-3.434-.528-3.643-.082-.29-.363-.41-.631-.27-.22.112-.234.278-.102 1.166.049.33.078.611.064.624-.013.012-.303.06-.643.106a22.2 22.2 0 00-.697.102c-.073.017-.082-.016-.162-.62-.102-.769-.128-.869-.255-.996-.115-.115-.326-.133-.502-.042-.141.073-.193.174-.193.373 0 .29.498 3.565.563 3.702.043.084.11.153.194.197.16.08.192.08.355.012zm-4.409-.708c.737-.25 1.332-.924 1.517-1.717.245-1.047-.27-2.087-1.21-2.44-.543-.206-1.021-.177-1.57.092-.74.362-1.235 1.14-1.282 2.01-.04.78.277 1.442.878 1.827.486.311 1.155.403 1.667.228zm-1.015-.806c-.222-.078-.533-.39-.594-.596-.196-.664.217-1.672.806-1.967.508-.254 1.18-.018 1.4.491.272.634-.166 1.758-.803 2.057-.206.097-.558.104-.809.015zm9.267 1.277l-.01-.742-.094.275c-.069.206-.1.258-.12.207-.015-.038-.173-1.11-.352-2.385l-.324-2.317.18-.51a5.8 5.8 0 01.188-.499c.006.005.121.888.257 1.962.135 1.073.252 1.977.26 2.008.008.03.016-.582.019-1.362l.004-1.417h5.145v5.52H18.03zm3.045.086c.356-.095.611-.245.889-.525.438-.44.628-.907.628-1.542 0-.588-.18-1.068-.553-1.478a1.897 1.897 0 00-1.474-.65c-1.072-.002-1.87.706-2.042 1.81-.163 1.057.492 2.116 1.472 2.38.263.07.824.073 1.08.005zm-.913-.79c-.194-.07-.313-.16-.462-.354-.178-.232-.25-.515-.247-.96.007-.836.4-1.313 1.082-1.313.363 0 .575.087.788.324.247.274.333.578.315 1.113-.013.366-.026.427-.137.654-.14.283-.315.45-.566.534a1.44 1.44 0 01-.773.002zM.373 13.723c-.388-2.37-.393-2.413-.317-2.438.133-.043 4.904-.757 4.924-.736.01.01.178 1.065.373 2.343s.36 2.36.368 2.401c.013.075-.078.092-2.45.461-1.353.211-2.471.383-2.481.383-.012 0-.2-1.087-.417-2.414zm2.858 1.7c.97-.167 1.164-.222 1.277-.366a.368.368 0 00-.133-.553c-.176-.084-.36-.073-1.175.07l-.726.126.025-.1c.044-.18.27-.543.694-1.112.635-.853.835-1.34.766-1.87-.053-.416-.162-.552-.443-.552-.295 0-1.934.294-2.11.378-.228.11-.297.305-.184.524.117.225.301.237 1.23.08l.63-.107v.114c0 .175-.12.38-.495.851-.677.85-.955 1.372-.997 1.88-.025.298.034.58.148.702.118.126.453.111 1.493-.066zm20.055-2.087v-2.744l.347-.39.348-.39.01 2.776.009 2.777-.357.357-.357.357zm-17.843-.5l-.361-2.366.16-.8c.087-.44.161-.776.164-.745.012.14.451 3.014.462 3.024.006.007.371-.844.81-1.89.44-1.048.813-1.926.83-1.952.039-.063.47 1.354.446 1.467-.01.046-.438 1.03-.95 2.187l-.934 2.103-.133.668-.133.67zM0 11.15c0-.015.07-.393.157-.838l.159-.81 2.499-.4c.834-.135 1.668-.265 2.503-.39.009.013-.32 1.69-.335 1.704-.014.013-4.871.762-4.947.762-.019 0-.035-.012-.036-.028zm18.2-.732c.056-.067.224-.253.373-.413l.27-.292 2.543.01 2.543.01-.363.403-.362.404h-5.107z"/></svg>
```

Comment:
Hello
Thanks for the PR!
Also have a close look into our [CONTRIBUTING guidlines](https://github.com/simple-icons/simple-icons/blob/develop/CONTRIBUTING.md) and here to our standards. (the svg is not what the linter expects, thus the build is failing currently) (icon name and icon position)